src/services/UserGroupService.js

```diff
@@ -44,3 +44,46 @@ export async function findByUserId(userId) {
     .then(({ data }) => data.results)
     .catch(manageError);
 }
+
+/**
+ * Add user to the "users" field of the group.
+ * @param {string} userId - User id.
+ * @param {string} groupId - Group id.
+ * @returns {Promise<object>} Promise with nothing on success otherwise an error.
+ */
+async function addUserToGroup(userId, groupId) {
+  return api.put(
+    `/api/classes/Group/${groupId}`,
+    {
+      users: {
+        __op: 'AddRelation',
+        objects: [
+          {
+            __type: 'Pointer',
+            className: '_User',
+            objectId: userId,
+          },
+        ],
+      },
+    },
+    { headers: getDefaultHeaders() },
+  );
+}
+
+/**
+ * Attach groups to user.
+ * @param {string} userId - User id.
+ * @param {string[]} groupIds - Array of group id.
+ * @returns {Promise<object>} Promise with nothing on success otherwise an error.
+ */
+export async function attachGroups(userId, groupIds) {
```

Comment:
move all in UserService and keep its name.

Reply:
okay